src/main/java/com/bathanh/apibook/api/book/BookItemDTO.java

```diff
@@ -0,0 +1,13 @@
+package com.bathanh.apibook.api.book;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
```

Comment:
Nó có empty constructor sẵn rồi nên không cần nữa

```suggestion
@NoArgsConstructor
```